### PR TITLE
Merge the Core TCK and the TCK.

### PR DESCRIPTION
**Pull request by cescoffier**
Having a single TCK is more inline with the other MP spec structure.

As the Core TCK covers most of the TCK content, some content has been deleted.

### CODE REVIEW

File: tck/README.adoc [hunk 1]
```diff
@@ -16,31 +16,53 @@
...
 
-== Running the TCK against an implementation
+In your implementation, create a new module depending on the TCK artifact:
 
-The TCK uses http://arquillian.org/[Arquillian], so an implementation needs Arquillian container support to run. In addition, the implementation must also implement the TCK's SPI in order to allow the TCK to interact and make assertions on messages sent and received from topics under test.
+[source,xml]
+----
+<dependency>
+    <groupId>org.eclipse.microprofile.reactive.messaging</groupId>
+    <artifactId>microprofile-reactive-messaging-core-tck</artifactId>
```
Comment (hutchig) [inline tck/README.adoc]: should this just be tck now  and not core-tck?

Comment (cescoffier) [reply tck/README.adoc]: Absolutely!

File: tck/README.adoc [hunk 2]
```diff
@@ -16,31 +16,53 @@
...
+    <version>${microprofile-reactive-messaging.version}</version>
+</dependency>
+----
 
-=== `TckContainer`
+This project will likely also depends on your implementation.
```
Comment (hutchig) [inline tck/README.adoc]: Suggest "Your test project that runs the TCK is also very likely to depend on your implementation project."

File: tck/README.adoc [hunk 3]
```diff
@@ -16,31 +16,53 @@
...
 
-The primary entry point into the tck is the `org.eclipse.microprofile.reactive.messaging.tck.spi.TckContainer`. An instance of this must be provided through the JDK `ServiceLoader` mechanism (that is, provide a `META-INF/services/org.eclipse.microprofile.reactive.messaging.tck.spi.TckContainer` file that contains the fully qualified classname of the implementation).
+As the tests are implemented in the TCK artifacts, you need to configure your test running to scan the tests included
```
Comment (hutchig) [inline tck/README.adoc]: running -> run

File: tck/README.adoc [hunk 4]
```diff
@@ -16,31 +16,53 @@
...
+in this artifacts. With Maven it's done using the following plugin configuration:
```
Comment (hutchig) [inline tck/README.adoc]: artifacts -> artifact

File: tck/README.adoc [hunk 5]
```diff
@@ -16,31 +16,53 @@
...
+[source,xml]
+----
+<build>
+  <plugins>
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-surefire-plugin</artifactId>
+      <version>2.22.1</version>
+      <configuration>
+        <dependenciesToScan>org.eclipse.microprofile.reactive.messaging:microprofile-reactive-messaging-core-tck</dependenciesToScan>
```
Comment (hutchig) [inline tck/README.adoc]: is this "core" still there?

Comment (cescoffier) [reply tck/README.adoc]: already fixed ;-)

**Comment by cescoffier** [comment]
Thanks for the very quick review! All comments have been handled. Merging.

**Comment by hutchig** [comment]
Fine.

**Pull request merged**